react/src/components/organisms/PageFlipper/PageFlipper.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, text } from '@storybook/addon-knobs/react';
+import { ArrowNav, DecorativeLink } from '../../../index';
+
+import PageFlipper from './index';
+import PageFlipperReadme from './PageFlipper.md';
+
+storiesOf('organisms', module)
+  .addDecorator(withKnobs)
+  .add('PageFlipper', withInfo({ PageFlipperReadme })(() => {
+    const decorativeLinkProps = {
+      href: text('Decorative href', ''),
```

Comment:
same as above, can you update these knob labels to match other components e.g.`PageFlipper.context.href`